modules/about-must-gather.adoc

```diff
@@ -23,12 +23,10 @@ Alternatively, you can collect specific information by running the command with
 
 * To collect data related to one or more specific features, use the `--image` argument with an image, as listed in a following section.
 +
-For example:
-+
+.Example command
 [source,terminal,subs="attributes+"]
 ----
-$ oc adm must-gather \
-  --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion}
+$ oc adm must-gather --image=registry.redhat.io/openshift-gitops-1/must-gather-rhel8:v1.10.0
 ----
 
 * To collect the audit logs, use the `-- /usr/bin/gather_audit_logs` argument, as described in a following section.
```

Comment:
Below is one more "For example:" that should be changed to ".Example command"

And after that is another "For example:", that one does not represent command though. It seems to be the list of the created pods. So I would change it to something like: ".Example pods" .

Reply:
Looks good to me, thank you!